ebpfcore/ebpf_drv.c

```diff
@@ -377,6 +377,7 @@ DriverEntry(_In_ DRIVER_OBJECT* driver_object, _In_ UNICODE_STRING* registry_pat
     NTSTATUS status;
     WDFDRIVER driver_handle;
     WDFDEVICE device;
+    bool ebpf_trace_initiated = false;
```

Comment:
For my understanding, Do you know if this change is relevant to this PR? Or was it observed during this change and needed to be fixed?

Reply:
I can move it to its own PR.

Reply:
It is okay to have here. 
Just for tracking, can you paste the relevant issue number?

Reply:
https://github.com/microsoft/ebpf-for-windows/issues/2838